CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ FEATURES:
 
 IMPROVEMENTS:
 
+* Add support for vCD 10.0 (API 33)
```

Comment:
I think I wouldn't mention the API 33 as it may lead to false conclusion that we use API v33.

Reply:
fixed